lib/AppInfo/Application.php

```diff
@@ -58,6 +60,7 @@ public function register(IRegistrationContext $context): void {
 		$context->registerEventListener(LoadAdditionalScriptsEvent::class, LoadAdditionalScriptsListener::class);
 		$context->registerEventListener(ShareCreatedEvent::class, ShareAutoAcceptListener::class);
 		$context->registerEventListener(BeforeTemplateRenderedEvent::class, TalkIntegrationListener::class);
+		$context->registerEventListener(BeforeUserManagementRenderedEvent::class, BeforeUserManagementRenderedListener::class);
```

Comment:
I think you'll need to add the interface to https://github.com/nextcloud/guests/blob/e671b06a1b911db2bbdb97e704be6d7c409d1f85/tests/stub.php